includes/restoreMappings.js

```diff
@@ -17,6 +17,7 @@ const { BackupError } = require('./error.js');
 const debug = require('debug');
 
 const mappingDebug = debug('couchbackup:mappings');
+const RESUME_COMMENT = '##RESUME##';
```

Comment:
I've been wondering about whether we can make this comment in some kind of standards compliant way so that adding this robustness to the backup file doesn't necessarily make it incompatible with other parsers.
For example [JSON text sequences - RFC7464](https://datatracker.ietf.org/doc/html/rfc7464) provides a way to detect truncation using the `\u001E` record separator and `\n`.

We could "formalize" json-seq compatibility in the backup file by adding `\u001E` at the start of backup lines with this change. As whitespace I think it wouldn't break existing parsing (broken lines are already broken). This would also give us a way to detect that the backup line came from a version before this change, such that we could handle backwards compatibility.

This would also eliminate the (admittedly extremely low) chance of bumping into broken JSON that happens to match the comment because the `\u001E` would need to be escaped within any JSON content.

```suggestion
const RESUME_COMMENT = `\u001E${JSON.stringify('##@cloudant/couchbackup:RESUME##')}`;
```

Of course test fixtures will need updating if we go this route.